pgd_splicer/ProcessPDBTask.py

```diff
@@ -403,6 +403,58 @@ def atom_noamino(s):
     return s.startswith("ATOM ") and not amino_present(s)
 
 
+class OccupancySelect(Select):
+    """
+    This class implements the occupancy awareness selection.
+
+    Each disordered residue contains one or more disordered atom structures.  These structures contain one or more atoms, each with its own altloc and occupancy values.  As each atom associated with a particular altloc has its own occupancy value, the altloc with the highest average occupancy value is identified.  Atoms with other altloc values are removed from the structure.
```

Comment:
Heh, I had no idea. The actual source of my gripe was that GH was cutting off the line so I couldn't read the whole thing. Thanks for changing it though :-)